device_operation/include/device_gemm_xdl_c_shuffle.hpp

```diff
@@ -29,22 +29,23 @@ template <
     ck::index_t BlockSize,
     ck::index_t MPerBlock,
     ck::index_t NPerBlock,
-    ck::index_t K0PerBlock,
-    ck::index_t K1,
+    ck::index_t KPerBlock,
     ck::index_t MPerXDL,
     ck::index_t NPerXDL,
     ck::index_t MXdlPerWave,
     ck::index_t NXdlPerWave,
     typename ABlockTransferThreadClusterLengths_K0_M_K1,
     typename ABlockTransferThreadClusterArrangeOrder,
     typename ABlockTransferSrcAccessOrder,
+    ck::index_t ABlockTransferK1PerBlock,
```

Comment:
I think it should be A_K1 and B_K1

Reply:
How about AK1PerBlock/BK1PerBlock? I am trying to follow naming consistency

Reply:
AK1 is length of grid tensor, so its name should not include "PerBlock"

Reply:
I see what you mean. Will fix that up